src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,30 @@
 import React from 'react';
+import PropTypes, { string } from 'prop-types';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard
+        {...movie}
+        key={movie.imdbId}
+      />
+    ))}
+  </div>
 );
+MoviesList.propTypes = {
+  movies: PropTypes.arrayOf(
+    PropTypes.shape({
+      title: string.isRequired,
+      imgUrl: string.isRequired,
+      imdbUrl: string.isRequired,
+      imdbId: string.isRequired,
+      description: string,
```

Comment:
add description to defaultProps or add isRequired to it

Reply:
also, let's move MovieType to the separate file, so you can reuse these proptypes in both MovieList and MovieCard
like in this video
https://www.youtube.com/watch?v=XauMMs5XSnI&feature=youtu.be